packages/nocodb/src/lib/meta/api/sync/helpers/job.ts

```diff
@@ -577,9 +577,16 @@ export default async (
             ncCol.colOptions = {
               options: [...colOptions.data],
             };
-            // if options are empty, configure '' as default option
-            ncCol.dtxp =
-              colOptions.data.map((el) => `'${el.title.replace(/'/gi, "''")}'`).join(',') || "''";
+
+            if (['mysql', 'mysql2'].includes(getRootDbType())) {
+              if (colOptions.data.length > 63) {
```

Comment:
enum can have more than 64 options. Need discussion on the upper bound. (taking how we display in UI into consideration as well)

e.g.

enum('51','7','35','19','26','4','18','20','16','61','30','22','37','38','14','50','23','48','34','17','32','5','9','15','45','42','54','53','52','28','29','43','40','0','56','33','55','39','58','1','3','31','62','49','6','2','57','63','60','36','44','65','64','8','25','47','46','59','13','41','10','21','12','11','24','27','66')


cc: @o1lab